web/src/lib/utils/helpers.tsx

```diff
@@ -310,3 +310,11 @@ export const makeButtonIcon = (svgFilename: string, size = "20px"): ReactElement
     alt=""
   />
 );
+
+export const isIndustryIdGeneric = (industry: Industry) => {
+  return industry.id === "generic";
+};
+
+export const getIndustryName = (industry: Industry) => {
+  return industry.name;
+};
```

Comment:
I don't think this helper adds any value to the code, since it is just returning something we already have access to.

Reply:
I removed the function and passed the `name` key as an argument to the orderBy function